core/src/test/resources/UsageTestData.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "valid": "true",
+    "payload": "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6IklMX04xYlBjUlA0alpVdzJlWjNyODd5YXZkVSIsImtpZCI6IklMX04xYlBjUlA0alpVdzJlWjNyODd5YXZkVSJ9.eyJhdWQiOiJodHRwczovL2FuYWx5c2lzLndpbmRvd3MtaW50Lm5ldC9wb3dlcmJpL2FwaSIsImlzcyI6Imh0dHBzOi8vc3RzLndpbmRvd3MtcHBlLm5ldC8wNDEyNWEzNS1kZmJhLTQ5YTEtYjRmZC05OGM0NzNiZWIwNjUvIiwiaWF0IjoxNjkxMTY3MTgxLCJuYmYiOjE2OTExNjcxODEsImV4cCI6MTY5MTE3MTEwOSwiYWNjdCI6MCwiYWNyIjoiMSIsImFpbyI6IkFUUUF5LzhYQUFBQXFydkEvTG5vSTh4MUZEMUZtYjcxVHBIOUtMVU44N21xYTJCc1Z0NlNHVlRqaGlQcGhIeU5SQ3VjZXJ1TmUvSnQiLCJhbXIiOlsicHdkIl0sImFwcGlkIjoiMDAwMDAwMDktMDAwMC0wMDAwLWMwMDAtMDAwMDAwMDAwMDAwIiwiYXBwaWRhY3IiOiIyIiwiZmFtaWx5X25hbWUiOiJFZG9nIiwiZ2l2ZW5fbmFtZSI6IkFkbWluIiwiaXBhZGRyIjoiMjYwMTo2NDY6YzYwMDo4YTQwOjQ4OWU6MWM1Zjo4MDYwOjk2NmQiLCJuYW1lIjoiRWRvZyBBZG1pbnMiLCJvaWQiOiIzMzA0MThjZi03YTU3LTQxNzEtOTA4Mi1kNzE2MTg5OTBmNjEiLCJwdWlkIjoiMTAwM0RGRkQwMTk5RjY2RCIsInJoIjoiMC5BQUVBTlZvU0JMcmZvVW0wX1pqRWM3NndaUWtBQUFBQUFBQUF3QUFBQUFBQUFBQUJBSE0uIiwic2NwIjoidXNlcl9pbXBlcnNvbmF0aW9uIiwic3ViIjoiYVl3cTFadERSSEtlQ011QU1kOGI0bVE0NjlRRTZscmZKcDJ2aXpMM2dRMCIsInRpZCI6IjA0MTI1YTM1LWRmYmEtNDlhMS1iNGZkLTk4YzQ3M2JlYjA2NSIsInVuaXF1ZV9uYW1lIjoiRWRvZ0FkbWluQFRyaWRlbnRFZG9nLmNjc2N0cC5uZXQiLCJ1cG4iOiJFZG9nQWRtaW5AVHJpZGVudEVkb2cuY2NzY3RwLm5ldCIsInV0aSI6IjB6Nmt6ajhrNTBpNHp4OVh4WkFEQUEiLCJ2ZXIiOiIxLjAiLCJ3aWRzIjpbIjYyZTkwMzk0LTY5ZjUtNDIzNy05MTkwLTAxMjE3NzE0NWUxMCIsImI3OWZiZjRkLTNlZjktNDY4OS04MTQzLTc2YjE5NGU4NTUwOSJdLCJ4bXNfcGwiOiJlbi1VUyJ9.OhNT7-dNRzH7JJpPtfs8f-TX1JmBYNrlePHOpQXVJq9U2MiFBougw8ea-DjRaE3lAIaBqYr9aKKlXdH_fnFF_d-OZ3REmFNPLASMKR-n8mUOQWBXwp8eaeL2A8IAD5GJkjjeu9VeTWr5ApI_AnE-gPfJbwQKpxbkSJbTcRQfVKGbaeOH_9iw5rrndUDZNURZeSW4UfS0X3SjBKQSE-J69c0yPinhZPwEQEpSvEbMIWmpHItKCew5URCuEj3cv6DU6lVEYeGlfzgJ5BTjhlW6imxOV2Ed-78pYZNmr-LcfSExa6CPtuQhYFdYCOuMVDgP53z1_8RTTy_bxb-KxfC4VQ"
```

Comment:
Are these OK to be made public

Reply:
Those are dummy values. Can't be used to make rest calls. But not sure how tools that automatically glean our code would react. I had similar question in mind.

Reply:
Ok removed the dummy token from the repo. Created a dummy token generator instead.